src/components/atoms/Input/Input.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+
+import { text, boolean, number, select } from '@storybook/addon-knobs/react'
+import { storiesOf } from '@storybook/react'
+import { wInfo } from 'stories/index.stories'
+import { Input } from './Input'
+
+storiesOf('Components/Text', module).addWithJSX(
```

Comment:
Components/Test it's ok? I think maybe we can move to Components/Forms, What do you think?